libs/components/code-examples/src/lib/modules/filter-bar/filter-bar/basic/filter-selection-values.ts

```diff
@@ -0,0 +1,81 @@
+import { SkyFilterBarFilterValue } from '@skyux/filter-bar';
+
+// eslint-disable @cspell/spellchecker
+export const FILTER_SELECTION_VALUES: Record<
+  string,
+  SkyFilterBarFilterValue[]
+> = {
+  'community-connection': [
+    { value: 'child-of-faculty', displayValue: 'Child of faculty' },
+    { value: 'child-of-alum', displayValue: 'Child of alum' },
+    { value: 'grandparent-is-alum', displayValue: 'Grandparent is an alum' },
```

Comment:
```suggestion
    { value: 'grandparent-is-alum', displayValue: 'Grandchild of alum' },
```
I'd tweak for parallel construction with "Child of alum"